app/controllers/manageiq/api/base_controller/blueprints.rb

```diff
@@ -42,17 +42,33 @@ def update_bundle(blueprint, bundle)
         def deserialize_bundle(bundle)
           options = {}
           if bundle["service_catalog"]
```

Comment:
If this reads:

```ruby`
if bundle.key?("service_catalog")
```

then I think the setting it to `nil` part should work :smile: